### PR TITLE
Add local generator packages for plantuml

### PR DESCRIPTION
**Pull request by hollowdrutt**
The description of adding packages to Dockerfile for using local techdoc generator does not include dependencies needed when documentation contains plantuml diagram. 
Added Dockerfile commands based on the techdocs container https://github.com/backstage/techdocs-container/blob/main/Dockerfile

### CODE REVIEW

**Review by ahhhndre — CHANGES_REQUESTED** [review]
Hi @hollowdrutt, thanks for the contribution!

Would it make sense to update this to use the more recent version? There looks to be two for 2023 already: http://sourceforge.net/projects/plantuml/files/

**Comment by ahhhndre** [comment]
I’ve submitted an issue for the TechDocs container as well seeing as that was the source of your updates: https://github.com/backstage/techdocs-container/issues/46

**Comment by hollowdrutt** [comment]
@ahhhndre thanks for the feedback, I have updated the version of plantuml.

File: docs/features/techdocs/getting-started.md [hunk 1]
```diff
@@ -270,8 +270,11 @@ You can do so by including the following lines right above `USER node` of your
 `Dockerfile`:
 
 ```Dockerfile
-RUN apt-get update && apt-get install -y python3 python3-pip
+RUN apt-get update && apt-get install -y python3 python3-pip gcc musl-dev openjdk-17-jdk curl graphviz fonts-dejavu fontconfig
```
Comment (andrewthauer) [inline docs/features/techdocs/getting-started.md:273]: Might be nice to make the plantuml version a variable with some instructions on how to bump it. These are the partial parts of the custom techdocs Dockerfile we use. On that note and improvement to the instructions would be to have one line script that can get the SHA value.

```Dockerfile
# To update plantuml version:
#   1. Check the latest version from https://plantuml.com/download
#   2. Download the updated version from http://sourceforge.net/projects/plantuml/files/plantuml.[PLANTUML_VER].jar/download
#   3. Run `sha1sum plantuml.[PLANTUML_VER].jar` to get the checksum on the file you downloaded
#   4. Update the version & checksum args
ARG PLANTUML_VER="1.xxxx.x"
ARG PLANTUML_CHECKSUM="xxxxxxxx"

# ...

# Download plantuml file, Validate checksum & move plantuml file
RUN curl -o plantuml.jar -L "http://sourceforge.net/projects/plantuml/files/plantuml.$PLANTUML_VER.jar/download" \
  && echo "$PLANTUML_CHECKSUM  plantuml.jar" | sha1sum -c - \
  && mv plantuml.jar /opt/plantuml.jar
```

Comment (andrewthauer) [reply docs/features/techdocs/getting-started.md:273]: So, if we wanted to switch to using GH releases, the following bash will get the latest version/checksum:

```sh
VERSION="$(curl --silent "https://api.github.com/plantuml/plantuml/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' | sed -E 's/v([0-9.]+).*/\1/')"
CHECKSUM="$(curl -sSL "https://github.com/plantuml/plantuml/releases/download/v$VERSION/plantuml-$VERSION.jar" | sha1sum | cut -d ' ' -f 1)"
echo "PlantUML (Latest): $VERSION (checksum $CHECKSUM)"
```

Comment (hollowdrutt) [reply docs/features/techdocs/getting-started.md:275]: @andrewthauer thanks for the input! I have updated the PR.

**Review by andrewthauer — APPROVED** [review]

**Comment by Rugvip** [comment]
Nudge @backstage/techdocs-core 😁

**Pull request closed without merging**